server.go

```diff
@@ -1188,6 +1188,8 @@ func (s *Server) applyCreateShardGroupIfNotExists(m *messaging.Message) (err err
 		// Generate an ID for each shard.
 		for _, sh := range g.Shards {
 			sh.ID = tx.nextShardID()
+			sh.stats = NewStats(fmt.Sprintf("shard %d", sh.ID))
+			sh.stats.Inc("create")
```

Comment:
This is only for debugging.  We have an issue where the shard store is closed, and I need to find out if it's shards we open from disk, or create from memory.